app/services/bulk_upload/lettings/year2023/csv_parser.rb

```diff
@@ -61,6 +63,12 @@ def too_many_columns?
     max_columns_count > MAX_COLUMNS
   end
 
+  def correct_template_for_year?
+    collection_start_year_for_date(first_record_start_date) == 2023
```

Comment:
We could extract the year here to allow us to easily duplicate these files for the next year?

```suggestion
    collection_start_year_for_date(first_record_start_date) == FORM_YEAR
```

Reply:
Can do!